packages/core/admin/admin/src/content-manager/components/CollectionTypeFormWrapper/index.js

```diff
@@ -294,6 +294,29 @@ const CollectionTypeFormWrapper = ({ allLayoutData, children, slug, id, origin }
     ]
   );
 
+  const onDraftRelationCheck = useCallback(async () => {
+    try {
+      trackUsageRef.current('willCheckDraftRelations');
+
+      const endPoint = getRequestUrl(
+        `collection-types/${slug}/${id}/actions/numberOfDraftRelations`
+      );
+      dispatch(setStatus('draft-relation-check-pending'));
+
+      const numberOfDraftRelations = await axiosInstance.get(endPoint);
+      trackUsageRef.current('didCheckDraftRelations');
+
+      dispatch(setStatus('resolved'));
+
+      return Promise.resolve(numberOfDraftRelations);
```

Comment:
I think since you are using an `async` function it will always return a Promise and therefore you can simply `return numberOfDraftRelations`.

Also: can we return `numberOfDraftRelations.data` here?